src/group-configurations/content-groups-section/index.jsx

```diff
@@ -1,34 +1,85 @@
 import PropTypes from 'prop-types';
 import { useIntl } from '@edx/frontend-platform/i18n';
-import { Button } from '@edx/paragon';
+import { Button, useToggle } from '@edx/paragon';
 import { Add as AddIcon } from '@edx/paragon/icons';
 
 import GroupConfigurationContainer from '../group-configuration-container';
+import ContentGroupContainer from './ContentGroupContainer';
 import EmptyPlaceholder from '../empty-placeholder';
 import messages from './messages';
 
-const ContentGroupsSection = ({ availableGroup: { groups, name } }) => {
+const ContentGroupsSection = ({
+  availableGroup,
+  groupConfigurationsActions,
+}) => {
   const { formatMessage } = useIntl();
+  const [isNewGroupVisible, openNewGroup, hideNewGroup] = useToggle(false);
+  const { id: parentGroupId, groups, name } = availableGroup;
+  const groupNames = groups?.map((group) => group.name);
+
+  const contentGroupObject = (groupName) => ({
```

Comment:
[question]: Can we move this function into utilities?

Reply:
moved and renamed